pom.xml

```diff
@@ -79,55 +127,73 @@
 			<groupId>javax.jcr</groupId>
 			<artifactId>jcr</artifactId>
 			<version>2.0</version>
-            <optional>true</optional>
```

Comment:
Please revert this whitespace-only change, it makes checking the history more difficult.